### PR TITLE
Fix some eye bleeding sentences.

### PR DESCRIPTION
**Pull request by gabriel-tessier**

### CODE REVIEW

File: README-fr.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+> Apprendre a concevoir des systèmes à grande échelle.
 >
-> Préparation pour les entretiens de conception de systèmes.
+> Préparation pour les entretiens.
 
-### Apprendre a conceptionner des systèmes à grande echelle
+### Apprendre a concevoir des systèmes à grande échelle
 
-Apprendre à conceptionner des systèmes à grande echelle vous permettra de devenir une meilleur ingénieur.
+Apprendre à concevoir des systèmes à grande échelle vous permettra de devenir un(e) meilleur ingénieur(e).
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
Apprendre à concevoir des systèmes à grande échelle vous permettra de devenir un(e) meilleur(e) ingénieur(e).
```

File: README-fr.md [hunk 2]
```diff
@@ -119,16 +119,16 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
 * [Load balancer](#load-balancer)
     * [Active-passive](#active-passive)
     * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
+    * [Couche 4 load balancing](#layer-4-load-balancing)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [Répartition de charge couche 4](#layer-4-load-balancing)
```
WDYT ?

File: README-fr.md [hunk 3]
```diff
@@ -119,16 +119,16 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
     * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
+    * [Couche 4 load balancing](#layer-4-load-balancing)
+    * [Couche 7 load balancing](#layer-7-load-balancing)
     * [Horizontal scaling](#horizontal-scaling)
 * [Reverse proxy (web server)](#reverse-proxy-web-server)
     * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Couche Application](#application-layer)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Couche Applicative](#application-layer)
```

**Review by spuyet — CHANGES_REQUESTED** [review]
Thanks for you contribution, I've added some comments though :smile:

File: README-fr.md (hunk 2 above)
Comment (gabriel-tessier) [reply README-fr.md]: Good!

File: README-fr.md (hunk 3 above)
Comment (gabriel-tessier) [reply README-fr.md]: better

File: README-fr.md (hunk 1 above)
Comment (gabriel-tessier) [reply README-fr.md]: I mess this one!

File: README-fr.md [hunk 4]
```diff
@@ -119,16 +119,16 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
 * [Load balancer](#load-balancer)
     * [Active-passive](#active-passive)
     * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
+    * [Répartition de charge couche 4](#layer-4-load-balancing)
+    * [Couche 7 load balancing](#layer-7-load-balancing)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [Répartition de charge couche 7](#layer-7-load-balancing)
```

File: README-fr.md [hunk 5]
```diff
@@ -119,16 +119,16 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
     * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
+    * [Répartition de charge couche 4](#layer-4-load-balancing)
+    * [Couche 7 load balancing](#layer-7-load-balancing)
     * [Horizontal scaling](#horizontal-scaling)
 * [Reverse proxy (web server)](#reverse-proxy-web-server)
     * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Couche Applicative](#application-layer)
```
Comment (spuyet) [inline README-fr.md:127]:
```suggestion
* [Couche application](#application-layer)
```
"application" seems to be the right word(https://fr.wikipedia.org/wiki/Couche_application) as you suggested before, I'm just removing the capital letter

**Comment by gabriel-tessier** [comment]
Need several read and re-read and re-re-read before having something readable... 
I'll check again.

**Comment by gabriel-tessier** [comment]
May help if this pull request is merged as it's 4 months old....

File: README-fr.md [hunk 6]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-        * [Denormalization](#denormalization)
-        * [SQL tuning](#sql-tuning)
+    * [Découverte de service](#service-discovery)
+* [Base de données](#database)
+    * [Systeme de Gestion Base de Données Relationel (SGBDR)](#relational-database-management-system-rdbms)
+        * [Réplication principale-secondaire](#master-slave-replication)
+        * [Réplication principale-principale](#master-master-replication)
+        * [Féderation](#federation)
+        * [Partitionement](#sharding)
+        * [Dénormalization](#denormalization)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
        * [Dénormalisation](#denormalization)
```

File: README-fr.md [hunk 7]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+        * [Optimisation SQL](#sql-tuning)
     * [NoSQL](#nosql)
-        * [Key-value store](#key-value-store)
-        * [Document store](#document-store)
+        * [Stockage Clé-Valeur](#key-value-store)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
        * [Stockage clé-valeur](#key-value-store)
```

File: README-fr.md [hunk 8]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+        * [Stockage de Document](#document-store)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
        * [Stockage de document](#document-store)
```

File: README-fr.md [hunk 9]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-    * [Web server caching](#web-server-caching)
-    * [Database caching](#database-caching)
-    * [Application caching](#application-caching)
-    * [Caching at the database query level](#caching-at-the-database-query-level)
-    * [Caching at the object level](#caching-at-the-object-level)
-    * [When to update the cache](#when-to-update-the-cache)
+    * [Mise en cache client](#client-caching)
+    * [RDC (CDN) de caching](#cdn-caching)
+    * [Cache du Serveur web](#web-server-caching)
+    * [Cache de Base de données](#database-caching)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [Cache de base de données](#database-caching)
```

File: README-fr.md [hunk 10]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-    * [CDN caching](#cdn-caching)
-    * [Web server caching](#web-server-caching)
-    * [Database caching](#database-caching)
-    * [Application caching](#application-caching)
-    * [Caching at the database query level](#caching-at-the-database-query-level)
-    * [Caching at the object level](#caching-at-the-object-level)
-    * [When to update the cache](#when-to-update-the-cache)
+    * [Mise en cache client](#client-caching)
+    * [RDC (CDN) de caching](#cdn-caching)
+    * [Cache du Serveur web](#web-server-caching)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [Cache du serveur web](#web-server-caching)
```

File: README-fr.md [hunk 11]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+    * [Injection RDC (CDNs)](#push-cdns)
+    * [Récupération RDC (CDNs)](#pull-cdns)
+* [Répartiteur de Charge](#load-balancer)
+    * [Actif-passif](#active-passive)
+    * [Actif-actif](#active-active)
+    * [Répartition de charge couche 4](#layer-4-load-balancing)
+    * [Répartition de charge couche 7](#layer-7-load-balancing)
+    * [Mise à l'échelle horizontale](#horizontal-scaling)
+* [Proxy inverse (serveur web)](#reverse-proxy-web-server)
+    * [Répartiteur de Charge vs Proxy inverse](#load-balancer-vs-reverse-proxy)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [Répartiteur de charge vs Proxy inverse](#load-balancer-vs-reverse-proxy)
```

File: README-fr.md [hunk 12]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+    * [Cohérence](#eventual-consistency)
+    * [Cohérence Forte](#strong-consistency)
+* [Patrons de Disponibilité](#availability-patterns)
+    * [Basculement](#fail-over)
+    * [Réplication](#replication)
+* [Système de Noms de Domaine](#domain-name-system)
+* [Réseau de diffusion de contenu](#content-delivery-network)
+    * [Injection RDC (CDNs)](#push-cdns)
+    * [Récupération RDC (CDNs)](#pull-cdns)
+* [Répartiteur de Charge](#load-balancer)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Répartiteur de charge](#load-balancer)
```

File: README-fr.md [hunk 13]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
+    * [Théorème CAP ou CDP](#cap-theorem)
+        * [CP - Cohérence et tolérance au Partitionnement](#cp---consistency-and-partition-tolerance)
+        * [DP - Disponibilité et tolérance au Partitionnement](#ap---availability-and-partition-tolerance)
+* [Patrons de Cohérence](#consistency-patterns)
+    * [Faible cohérence](#weak-consistency)
+    * [Cohérence](#eventual-consistency)
+    * [Cohérence Forte](#strong-consistency)
+* [Patrons de Disponibilité](#availability-patterns)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Patrons de disponibilité](#availability-patterns)
```

File: README-fr.md [hunk 14]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
+    * [Théorème CAP ou CDP](#cap-theorem)
+        * [CP - Cohérence et tolérance au Partitionnement](#cp---consistency-and-partition-tolerance)
+        * [DP - Disponibilité et tolérance au Partitionnement](#ap---availability-and-partition-tolerance)
+* [Patrons de Cohérence](#consistency-patterns)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Patrons de cohérence](#consistency-patterns)
```

File: README-fr.md [hunk 15]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-    * [Horizontal scaling](#horizontal-scaling)
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
+    * [Théorème CAP ou CDP](#cap-theorem)
+        * [CP - Cohérence et tolérance au Partitionnement](#cp---consistency-and-partition-tolerance)
+        * [DP - Disponibilité et tolérance au Partitionnement](#ap---availability-and-partition-tolerance)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
        * [DP - Disponibilité et tolérance au partitionnement](#ap---availability-and-partition-tolerance)
```

File: README-fr.md [hunk 16]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-    * [Layer 7 load balancing](#layer-7-load-balancing)
-    * [Horizontal scaling](#horizontal-scaling)
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
+    * [Théorème CAP ou CDP](#cap-theorem)
+        * [CP - Cohérence et tolérance au Partitionnement](#cp---consistency-and-partition-tolerance)
```
Comment (spuyet) [inline README-fr.md:106]:
```suggestion
        * [CP - Cohérence et tolérance au partitionnement](#cp---consistency-and-partition-tolerance)
```

File: README-fr.md [hunk 17]
```diff
@@ -99,57 +99,57 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-    * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
-    * [Horizontal scaling](#horizontal-scaling)
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
+* [Latence vs haut débit](#latency-vs-throughput)
+* [Disponibilité vs Cohérence](#availability-vs-consistency)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Disponibilité vs cohérence](#availability-vs-consistency)
```

**Review by spuyet — CHANGES_REQUESTED** [review]
I've added some comments :wink:

File: README-fr.md (hunk 11 above)
Comment (gabriel-tessier) [reply README-fr.md]: Fixed in another commit https://github.com/spuyet/system-design-primer/pull/1/commits/537031afc5a4b2774e74e7a84a35f6a28a043b55
 as I can't edit in this viewer.

**Comment by gabriel-tessier** [comment]
Here the last commit I do in this branch, I'll start a new branch with more limited scope after the merge of this one.
In this branch I made change here and there without real plan.
Thought about one new branch for titles.
Then one by paragraph or group of paragraph as some are really small, it may help to translate more faster.
We can also use projects tabs (https://github.com/spuyet/system-design-primer/projects) to cut the file, it will be more easy to pickup task from here and create pull requests.

File: README-fr.md [hunk 18]
```diff
@@ -80,13 +80,13 @@ Libre à vous d'envoyer des pull requests pour aider à:
 * Ajouter des nouvelles sections
 * [Traduire](https://github.com/donnemartin/system-design-primer/issues/28)
 
-Les sections qui nécessitent des finitions sont placés dans la section [en développement](#under-development).
+Les sections qui nécessitent des améliorations sont placés dans la section [en développement](#under-development).
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
Les sections qui nécessitent des améliorations sont placées dans la section [en développement](#under-development).
```

File: README-fr.md [hunk 19]
```diff
@@ -99,80 +99,80 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
-* [Load balancer](#load-balancer)
-    * [Active-passive](#active-passive)
-    * [Active-active](#active-active)
-    * [Layer 4 load balancing](#layer-4-load-balancing)
-    * [Layer 7 load balancing](#layer-7-load-balancing)
-    * [Horizontal scaling](#horizontal-scaling)
-* [Reverse proxy (web server)](#reverse-proxy-web-server)
-    * [Load balancer vs reverse proxy](#load-balancer-vs-reverse-proxy)
-* [Application layer](#application-layer)
+* [Performance vs évolutivité](#performance-vs-scalability)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
* [Performance vs scalabilité](#performance-vs-scalability)
```

File: README-fr.md [hunk 20]
```diff
@@ -99,80 +99,80 @@ Lisez les [directives de contribution](CONTRIBUTING.md).
...
+    * [Cache des objets](#caching-at-the-object-level)
+    * [Quand mettre à jour le cache](#when-to-update-the-cache)
         * [Cache-aside](#cache-aside)
         * [Write-through](#write-through)
         * [Write-behind (write-back)](#write-behind-write-back)
         * [Refresh-ahead](#refresh-ahead)
 * [Asynchronism](#asynchronism)
-    * [Message queues](#message-queues)
-    * [Task queues](#task-queues)
+    * [File d'attente de message](#message-queues)
```
Comment (spuyet) [inline README-fr.md]:
```suggestion
    * [File d'attente de messages](#message-queues)
```

**Review by spuyet — APPROVED** [review]

**Review by spuyet — APPROVED** [review]

**Review by spuyet — CHANGES_REQUESTED** [review]
Here is my last suggestions, I tried to commit by myself but I can't as this is your repo

**Comment by gabriel-tessier** [comment]
All changes are commited.

**Review by spuyet — APPROVED** [review]
Good to me, thx!

**Pull request merged**